Dockerfile

```diff
@@ -1,40 +1,47 @@
-FROM ubuntu:16.04
-MAINTAINER Kane Blueriver <kxxoling@gmail.com>
+FROM ubuntu:18.04
+LABEL maintainer="Kane Blueriver <kxxoling@gmail.com>"
 
+ARG NB_USER=clojure
+ARG NB_UID=1000
+ENV HOME /home/${NB_USER}
 ENV NOTEBOOK_PATH $HOME/notebooks
 ENV PORT 8888
 ENV CLOJUPYTER_PATH $HOME/clojupyter
-ENV LEIN_ROOT=1
-
-RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 0xb8d7f7791716c8a4
-RUN echo "deb http://ppa.launchpad.net/mikegedelman/leiningen-git-stable/ubuntu trusty main" >> /etc/apt/sources.list
-RUN echo "deb-src http://ppa.launchpad.net/mikegedelman/leiningen-git-stable/ubuntu trusty main" >> /etc/apt/sources.list
-
-RUN apt-get update && apt-get install -yq python-setuptools \
-        python-dev \
-        build-essential \
-        curl \
-        default-jre \
-        leiningen
-
-RUN lein self-install
-
-# Install Jupyter
-RUN mkdir -p $NOTEBOOK_PATH
-RUN easy_install pip
-RUN pip install jupyter
+ENV LEIN_ROOT 1
+
+USER root
+RUN apt update && apt install -yq \
+        python-pip \
```

Comment:
Indention by 8 here?

Reply:
Differentiate which packets are to be installed from the top row command, same thing applies to the `useradd` command

Reply:
I mean it's 8 spaces this line, but a tab the next line.